README.md

```diff
@@ -629,7 +629,7 @@ make create
 
```

Comment:
I don't think this should be a separate commit. The changes to the documentation should be in the same commit(s) as the corresponding code changes. Same goes for UPGRADING.rst.